### DISCUSSION TITLE
Missing words and translation not applied

### DISCUSSION THREAD

**Discussion by moanatari** [dfint/.github#17] [Q&A] [answered]
Hello everyone!
I'm starting to try to translate or adapt for French.
To do some tests I modify locally the "dictionary.csv" file and I launch the game to see how it reacts.
I noticed that I can add missing words, for now only "she" and "dwarven" for example and they appear translated in the game.
But these words are not in transifex.
Is it possible to add words in transifex?
I also noticed that some translations are in the "dictionary.csv" file and also on the transifex site, but they do not apply in the game.
I think it is probably related to updates or other changes.
Is there anything to do in particular for this type of case?

Have you encountered this kind of behavior in other languages?

Merci beaucoup and have a nice day !

**Comment by insolor** [comment, marked as answer]
- "dwarven" exists in the objects.pot resource (as a part of `[NAME:dwarf:dwarves:dwarven]`), and it's not translated to French: https://app.transifex.com/dwarf-fortress-translation/dwarf-fortress-steam/translate/#fr/objects/445354214
- As for "she", I'm not sure why it's missing. I've found it in an old csv file, but it's missing on transifex. I'll check it.

> I also noticed that some translations are in the "dictionary.csv" file and also on the transifex site, but they do not apply in the game.

At the moment, with the current way of translation, this is nearly the most of what we can do. Some text we cannot intercept, and therefore it's impossible to apply the translation to it.

**Reply by insolor** [reply]
I've returned "she" and other 3-character words, they were removed by mistake.

**Reply by moanatari** [reply]
I didn't take the time to reply to thank you.
The new words have all been integrated and have already all been translated!
I created a discord for the French translation community.
Would it be possible to add it to the list ?
The invitation link for the discord: https://discord.gg/WgUPt8n8FS
Thank you very much and have a nice day !

**Reply by insolor** [reply]
@moanatari sure, done

**Comment by moanatari** [comment]
Oh gosh why didn't I see it!
I'm sorry, I haven't fully mastered the tool yet. But I'm starting to feel more comfortable!
When I do a search I now choose all the resources from 50.*.
A huge part has been automatically translated, and sometimes it doesn't fit so I play the game and when I see something that doesn't fit I try to see how it is entered in the translation. But here it is the message at the start of the game ha ha. "A Dwarven Outpost".

I added last night for Dwarven and I see this morning that it is indeed in the dictionary!
![image](https://github.com/user-attachments/assets/7804882d-8e62-4e34-a02d-7fd7d4763a90)

For the untranslated parts I totally understand. It's mostly in the descriptions of the dwarves, but it's not a big deal. It's not a problem for the game at the moment.
